server/api-service/lowcoder-server/src/test/java/org/lowcoder/api/authentication/GoogleAuthenticateTest.java

```diff
@@ -49,6 +49,11 @@ public class GoogleAuthenticateTest {
     private AuthenticationService authenticationService;
 
     @Test
+    public void testGoogleRegisterSuccess() {
+        assertTrue(true);
```

Comment:
Generally I would not use a test like this because it gives false hope that the functionality works.
Better approach is to keep the test as is and annotate it with **@Disabled** with a brief description:

`@Disabled("Disabled until properly refactored")`